t/test_DB_backend.pl

```diff
@@ -103,9 +102,10 @@ sub run_zonemaster_test_with_backend_API {
 my $test_history =
 	$engine->get_test_history( { frontend_params => $frontend_params_1, offset => $offset, limit => $limit } );
 diag explain( $test_history );
-ok( scalar( @$test_history ) == 2 );
-ok( $test_history->[0]->{id} eq '1' || $test_history->[1]->{id} eq '1' );
-ok( length($test_history->[0]->{id}) == 16 || length($test_history->[1]->{id}) == 16 );
+ok( scalar( @$test_history ) == 2 ), 'Two tests created';
```

Comment:
Parenthesis mismatch here.

Reply:
I would say *mis*placed rather than *mis*match. Corrected. The code was legal, but what came after comma was meaningless.

Reply:
yes you are totally right, I couldn't find that word when commenting yesterday ;)